src/runtime-rs/crates/resource/src/cpu_mem/mod.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//! CpuMemResource is a simple wrapper of a subset of cpuinfo, meminfo and potentially
+//! some other fields related to resource management. CpuMemResource is added since many
+//! components need to query the current cpu/mem state without TomlConfig. This can now be
+//! accessed through ResourceManager
+
+use std::sync::Arc;
+
+use kata_types::config::TomlConfig;
+
+use anyhow::{Context, Result};
+
+/// Cpu information wrapper
+#[derive(Default, Debug, Clone, Copy)]
+pub struct CpuResource {
+    /// The default vcpu # when boot
+    default_vcpu: i32,
+
+    /// Current number of vCPUs
+    /// when boot, the number is set to *default_vcpus*
+    current_vcpu: i32,
+    // TODO(shuoyu.ys): cpuset mapping relation
```

Comment:
Maybe we should delete this comment at this time 🙈

Reply:
I agree 😄